<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnreadUsers</class>
 <widget class="QWidget" name="UnreadUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>407</width>
    <height>304</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>消息查看列表</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    font-family: &quot;微软雅黑&quot;;
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">
QLineEdit,QTextEdit {
 	color: rgb(0, 0, 0);
    background-color: rgba(0, 0, 0, 0);
	min-height:22px;
	border:1px solid rgb(57, 209, 206);
	border-radius:2px;
}

QLabel {
    color: rgb(0, 0, 0);
}

#labelNickName {
    color: rgb(0, 0, 0);
}
#frame
{
	border:1px solid gray;
}
#frameLeft
{
	border:none;
	border-right:1px solid gray;
}
#frameRight
{
	border:none;
}

QListView
{
	border:none;

}

 QCheckBox {
      
  }

  QCheckBox::indicator {
     height:25px;
      width:35px;
  }

  QCheckBox::indicator:unchecked {
     
	image: url(:/image/resource/off.png);
  }


  QCheckBox::indicator:checked {
      
	image: url(:/image/resource/on.png);
  }

 

 
 </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frameLeft">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelUnread">
           <property name="text">
            <string>未读成员</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="listView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameRight">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelRead">
           <property name="text">
            <string>已读成员</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="listViewRead">
           <property name="styleSheet">
            <string notr="true">QTreeView::indicator:checked 
{ 

subcontrol-position:  right center;

}</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
